### PR TITLE
[API] write order email send test in api bundle

### PR DESCRIPTION
**Pull request by SirDomin**
| Q               | A
| --------------- | -----
| Branch?         | master
| Bug fix?        | no
| New feature?    | no
| BC breaks?      | no
| Deprecations?   | no
| Related tickets | 
| License         | MIT

### CODE REVIEW

**Review by arti0090 — APPROVED** [review]

File: src/Sylius/Bundle/ApiBundle/Tests/CommandHandler/SendOrderConfirmationEmailHandlerTest.php [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+use Symfony\Bundle\FrameworkBundle\Test\KernelTestCase;
+use Symfony\Component\Filesystem\Filesystem;
+use Symfony\Contracts\Translation\TranslatorInterface;
+
+final class OrderConfirmationResendTest extends KernelTestCase
+{
+    /**
+     * @test
+     */
+    public function it_resends_order_confirmation_email(): void
```
Comment (GSadee) [inline src/Sylius/Bundle/ApiBundle/Tests/CommandHandler/SendOrderConfirmationEmailHandlerTest.php:33]:
```suggestion
    public function it_sends_order_confirmation_email(): void
```

File: src/Sylius/Bundle/ApiBundle/Tests/CommandHandler/SendOrderConfirmationEmailHandlerTest.php [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+        $orderRepository->findOneByTokenValue('TOKEN')->willReturn($order);
+
+        $sendOrderConfirmationEmailHandler = new SendOrderConfirmationHandler(
+            $emailSender,
+            $orderRepository->reveal()
+        );
+
+        $sendOrderConfirmationEmailHandler(new SendOrderConfirmation('TOKEN'));
+
+        self::assertSame(1, $emailChecker->countMessagesTo('johnny.bravo@email.com'));
```
Comment (GSadee) [inline src/Sylius/Bundle/ApiBundle/Tests/CommandHandler/SendOrderConfirmationEmailHandlerTest.php:76]:
```suggestion
        self::assertSame($emailChecker->countMessagesTo('johnny.bravo@email.com'), 1);
```

**Review by GSadee — APPROVED** [review]

**Comment by GSadee** [comment]
The base of this pull-request was changed, you need fetch and reset your local branch 
if you want to add new commits to this pull request. **Reset before you pull, else commits
may become messed-up.**

Unless you added new commits (to this branch) locally that you did not push yet, 
execute `git fetch origin && git reset "decouple-send-order-confirmation"` to update your local branch.
 
Feel free to ask for assistance when you get stuck :+1:

**Review by pamil — APPROVED** [review]

**Review by AdamKasp — APPROVED** [review]

**Review by Zales0123 — APPROVED** [review]

**Comment by Zales0123** [comment]
Thanks, @SirDomin! :tada:

**Pull request merged**